governance/votes/Executive vote - April 26, 2021.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Template - [Executive Vote] Approve April 2021 Governance Cycle - April 26, 2021
+
+---
+# [Executive Proposal] Approve April 2021 Governance Cycle - April 26, 2021
+
+The Governance Facilitator(s) and the MakerDAO Smart Contracts Domain Team have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this Executive proposal passes the following **MIP status changes** will occur:
+
+### MIP Amendments and Declaration of Intent
+* Approve MIP4c2-SP7: MIP4 Amendments
+* Approve MIP4c2-SP8: MIP 9 Amendments
+* Approve MIP4c2-SP13: MIP0 Amendments
+* MIP13c3-SP10: Declaration of Intent - eurDai will move to Accepted Status
+
+### Content Production Core Unit
+* MKT-001, Content Production Core Unit Proposals will move to Accepted status.
+	* Mandate will move to Accepted Status.
+	* Budget of 44,375 DAI per month for three months will move to Accepted Status.
+	* Facilitator Onboarding of Seth Goldfarb will move to Accepted Status.
+
+### Growth Core Unit
+* GRO-001, Growth Core Unit Proposals will move to Accepted status.
+	* Mandate will move to Accepted Status.
+	* Budget of 126,117 DAI per month for three months will move to Accepted Status.
+	* Facilitator Onboarding of Nadia Alvarez will move to Accepted Status.
+
+### MakerDAO Shop Core Unit
+* MDS-001, MakerDAO Shop Core Unit Proposals will move to Accepted status.
+	* Mandate will move to Accepted Status.
+	* Budget of 0 DAI with 50/50 profit sharing will move to Accepted Status.
+	* Facilitator Onboarding of Coulter Mulligan & Mike Porcaro will move to Accepted Status.
+	* MakerDAO Shop Core Unit will be granted the exclusive right to the shop.makerdao.com domain.
+
+### Protocol Engineering Core Unit
+* PE-001, Protocol Engineering Core Unit Proposals will move to Accepted status.
+	* Mandate will move to Accepted Status.
+	* Budget of 510,000 DAI per month for one year will move to Accepted Status.
+	* Facilitator Onboarding of Derek Flossman will move to Accepted Status.
+	* 510,000 DAI will be distributed to the Protocol Engineering Core Unit Multisig for funding Core Unit expenses.
```

Comment:
Call out these distributions separately, imo, in combination with the above change.